src/app/components/server-charts/server-charts.component.ts

```
@@ -0,0 +1,791 @@
+        if(item.config[labelsField] && labels.indexOf(item.config[labelsField]) === -1) {
+          labels.push(item.config[labelsField]);
+        }
+        if((item.config[scaleField] || item.config[scaleField] === 0) && scales.indexOf(item.config[scaleField]) === -1) {
+          scales.push(item.config[scaleField]);
+        }
+      });
+
+
+      if(labels) {
+        labels.sort((a, b) => {
+          if(!isNaN(a) && !isNaN(b)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)